SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -41,6 +42,15 @@ void FSpatialGDKEditorLayoutDetails::ForceRefreshLayout()
 	}
 }
 
+namespace
+{
+	FSpatialGDKEditorCommandLineArgsManager& GetCommandLineArgsManager()
```

Comment:
we only use it in the `CustomizeDetails` function, right? would it make sense to just inline the 2 lines at the beginning of the `CustomizeDetails` method instead of creating a new one? That would also mean we don't have to call `GetModuleChecked` multiple times.